tests/ui/pattern/usefulness/empty-types.normal.stderr

```diff
@@ -253,43 +211,61 @@ LL +             Some(_) => todo!()
    |
 
 error: unreachable pattern
-  --> $DIR/empty-types.rs:208:13
+  --> $DIR/empty-types.rs:199:13
    |
 LL |             _ => {}
    |             ^
 
 error: unreachable pattern
-  --> $DIR/empty-types.rs:213:13
+  --> $DIR/empty-types.rs:204:13
    |
 LL |             _ => {}
    |             ^
 
 error: unreachable pattern
-  --> $DIR/empty-types.rs:218:13
+  --> $DIR/empty-types.rs:209:13
    |
 LL |             _ => {}
    |             ^
 
 error: unreachable pattern
-  --> $DIR/empty-types.rs:223:13
+  --> $DIR/empty-types.rs:214:13
    |
 LL |             _ => {}
    |             ^
 
 error: unreachable pattern
-  --> $DIR/empty-types.rs:229:13
+  --> $DIR/empty-types.rs:220:13
    |
 LL |             _ => {}
    |             ^
 
 error: unreachable pattern
-  --> $DIR/empty-types.rs:288:9
+  --> $DIR/empty-types.rs:279:9
    |
 LL |         _ => {}
    |         ^
 
+error: unreachable pattern
+  --> $DIR/empty-types.rs:282:9
+   |
+LL |         (_, _) => {}
+   |         ^^^^^^
+
+error: unreachable pattern
+  --> $DIR/empty-types.rs:285:9
+   |
+LL |         Ok(_) => {}
+   |         ^^^^^
+
+error: unreachable pattern
+  --> $DIR/empty-types.rs:286:9
+   |
+LL |         Err(_) => {}
+   |         ^^^^^^
+
```

Comment:
not a blocker to stabilization, but these error messages can be a little bit confusing. We might want to add more context to it rather than a generic message.

Reply:
On the unreachable ones? I was thinking we should have a better message on the ones that still go "Err() required"

Reply:
When the unreachable pattern is caused by a type being uninhabited, we could point at it and say it is uninhabited. Oh and also it might be nice to add a suggestion for it to be removed?

Reply:
> When the unreachable pattern is caused by a type being uninhabited, we could point at it and say it is uninhabited.

Good point, that would be helpful.

> Oh and also it might be nice to add a suggestion for it to be removed?

For what to be removed? The type?

Reply:
The match arm.

Reply:
Isn't that already what the "unreachable pattern" lint is saying?

Reply:
Oh you mean a structured suggestion? eah that makes sense